gradle/project.gradle.kts

```diff
@@ -175,7 +174,8 @@ tasks {
         onlyIf { !dirJdk.exists() }
         doFirst {
             println("Making JDK locally accessible...")
-            val jdkPath = "java.home".sysProp.takeIf { it.isNotBlank() }?.let { Paths.get(it) } ?: failIO { "Unable to find JDK" }
+            val jdkPath = "java.home".sysProp.takeIf { it.isNotBlank() }?.let { Paths.get(it) }
```

Comment:
I think the line is not so long to need wrapping. Detracts from readability. What do you think?

Reply:
Yes, I was aware. I do sometimes make small changes after auto-formatting, very rarely. This would be it.

Well, then I suggest adjusting `sysProp` extension to also call `takeIf { it.isNotBlank() }` and become nullable. It kinda makes sense. This would solve the problem right?

Reply:
Done

Reply:
Thx.